pkg/azuredx/table.go

```diff
@@ -245,19 +245,29 @@ var longConverter = data.FieldConverter{
 var decimalConverter = data.FieldConverter{
 	OutputFieldType: data.FieldTypeNullableFloat64,
 	Converter: func(v interface{}) (interface{}, error) {
-		var ai *float64
+		var af *float64
 		if v == nil {
-			return ai, nil
+			return af, nil
 		}
-		jN, ok := v.(string)
-		if !ok {
-			return nil, fmt.Errorf("unexpected type, expected String but got type %T with a value of %v", v, v)
+
+		jS, sOk := v.(string)
```

Comment:
Is there a case where it would be a string?

in `tableFromJSON` we use https://golang.org/pkg/encoding/json/#Decoder.UseNumber, so it should also be a json.Number I believe.

Reply:
When running it live we get it as a string. Not sure why but it is 🙈

Reply:
Try this query: `datatable (Value:decimal)[1.2,22.1111,2323323.3] | where Value > 2`

Reply:
Perhaps the test data is wrong and it is always a string? That would make sense for this type...